journal/models.py

```diff
@@ -1441,6 +1446,121 @@ class ThematicAreaJournal(Orderable, CommonControlField):
     )
 
 
+class DigitalPreservationAgency(CommonControlField):
+    name = models.TextField(
+        verbose_name=_("Name"),
+        blank=True,
+        null=True,
+    )
+    acronym = models.CharField(
+        max_length=64,
+        null=True,
+        blank=True,
+        verbose_name=_("Acronym"),
+    )
+    url = models.URLField(
+        blank=True,
+        null=True,
+    )
+
+    autocomplete_search_field = "name"
+
+    def autocomplete_label(self):
+        return str(self)
+
+    class Meta:
+        verbose_name = "Digitial Preservation Agency"
+        verbose_name_plural = "Digital Preservation Agencies"
+        unique_together = [("name", "acronym", "url",)]
+        indexes = [
+            models.Index(
+                fields=[
+                    "name",
+                ]
+            ),
+            models.Index(
+                fields=[
+                    "url",
+                ]
+            ),
+            
+        ]
+
+    @classmethod
+    def load(cls, user):
+        if not cls.objects.exists():
```

Comment:
@samuelveigarangel pode deixar sem a condição, pois se já existir atualiza, senão cria. Além disso, se por acaso a lista for atualizada, o load será executado.